### ISSUE TITLE
How to destroy tooltip's DOM element?

### ISSUE THREAD

**Issue by yoannmoinet** [Caged/d3-tip#91]
I'm using this (AMAZING) plugin in a popup context.
Each time I open a popup, it creates a new d3 chart, and when closed, it destroys and cleans everything.

But it seems that a new tooltip get created/appended to the DOM each time, without the possibility to remove it.

Do I have to find the element myself and remove it from the DOM once I close my popup?

It would be really useful to have a little more 'power' over the DOM element of the tooltip.
Because it is not accessible via the API.

**Comment by patorjk** [comment]
I'm also having a similar issue. Something like this would be good for those who are creating components.

I've got a web app with a tabbed interface where users open and close tabs that contain visualizations (that use d3tip). The visualizations are each encased in their own AngularJS directive. Each directive manages and sets up its visualization along with its tooltip. It'd be nice if when the directive was removed from the dom that I could just call a method on the d3tip that was created to also remove it from the dom, something like:

tooltip.destroy();
